srcpkgs/setzer/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'setzer'
+pkgname=setzer
+version=0.3.2
+revision=1
+archs="i686 x86_64"
```

Comment:
why is this here? I tried aarch64 and armv7l-musl and they build fine (didn't test if they work correctly).

Reply:
I can't test them neither. What should we write under archs if it build fine but we can't test if it actually works?

Reply:
You can test them for example using the command `xbps-src -a aarch64 pkg setzer`, but I don't see why it shouldn't build fine. Just remove this line and if travis fails, you can add it again with the proper architectures.

Reply:
Travis didn't fail, it should be ok now